src/RequestHandler/outofprocess/forwardinghandler.cpp

```diff
@@ -314,15 +310,7 @@ FORWARDING_HANDLER::OnExecuteRequestHandler()
     m_RequestStatus = FORWARDER_DONE;
 
     //disbale client disconnect callback
```

Comment:
typo "disbale" while you're fixing around it....

Reply:
done